src/components/ButtonPanel.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import Button from './Button';
+
+export default class ButtonPanel extends Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = {
+      buttonGroups: [
+        { buttons: ['AC', '+/-', '%', '/'] },
+        { buttons: ['7', '8', '9', 'X'] },
+        { buttons: ['4', '+5', '6', '-'] },
+        { buttons: ['1', '2', '3', '+'] },
+        { buttons: ['0', '.', '='] },
```

Comment:
- [ ] We can remove the `+` sign from the number 5

Reply:
Done.